gr-blocks/python/blocks/parse_file_metadata.py

```diff
@@ -75,12 +75,14 @@ def parse_header(p, VERBOSE=False):
         r = pmt.dict_ref(p, pmt.string_to_symbol("rx_time"), dump)
         secs = pmt.tuple_ref(r, 0)
         fracs = pmt.tuple_ref(r, 1)
-        secs = float(pmt.to_uint64(secs))
+        secs = pmt.to_uint64(secs)
         fracs = pmt.to_double(fracs)
-        t = secs + fracs
+        t = float(secs) + fracs
+        info["rx_secs"] = secs
```

Comment:
Maybe `rx_time_secs`?

Reply:
Thanks! That seems a better choice of a name. I'll push a commit changing this.